9.5/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -65,13 +74,31 @@ function check_env_vars() {
     [ ${#POSTGRESQL_USER}     -le 63 ] || usage "PostgreSQL username too long (maximum 63 characters)"
     [ ${#POSTGRESQL_DATABASE} -le 63 ] || usage "Database name too long (maximum 63 characters)"
     postinitdb_actions+=",simple_db"
+  elif check_cred_secret "/run/secrets/pgusers/user" && [ -v POSTGRESQL_DATABASE ]; then
```

Comment:
This looks like we limit the container for one user, _forever_?  Sounds like we start to define API and that should be defined with respects to future..  I tried to "veto" before, but nobody listened to me (so now we can create just one db with environment variables).  And this looks like similarly limited approach.